py-polars/Makefile

```
@@ -10,8 +10,12 @@ else
 	VENV_BIN=$(VENV)/bin
 endif
 
-venv:  ## Set up virtual environment
+venv:  ## Set up virtual environment and install requirements
 	python3 -m venv $(VENV)
+	$(MAKE) requirements
+
+.PHONY: requirements
+requirements: ## Install/refresh all project requirements
```

Actually, spotted one thing: `requirements` depends on `venv`. I tested this and it doesnt end up in an infinite loop or anything.

```suggestion
requirements: venv  ## Install/refresh all project requirements
```

---

Integrated 👍

---

In the end, it doesn't matter too much. But don't you think it's nicer that it will set up the `venv` for you, rather than error out?

---

True... why not indeed :)